openlibrary/plugins/openlibrary/js/merge-request-table/index.js

```diff
@@ -149,131 +143,155 @@ async function updateCommentsView(mrid, comment) {
  */
 function removeRow(row) {
     row.parentNode.removeChild(row)
+    const modal = document.getElementById("mr-close-request-modal");
+    const refreshButton = document.getElementById("mr-close-refresh-btn");
+    toggleRefreshModal(modal, refreshButton)
 }
 
+
 /**
- * Adds functionality for toggling visibility of older comments.
- *
- * @param {NodeList<HTMLElement>} elems Links that toggle comment visibility
- */
+ * Adds functionality for toggling visibility of the older comments.
+*
+* @param {NodeList<HTMLElement>} elems Links that toggle comment visibility
+*/
 export function initShowAllCommentsLinks(elems) {
     for (const elem of elems) {
         elem.addEventListener('click', function() {
-            toggleAllComments(elem)
+            toggleAllComments(elem);
```

Comment:
```suggestion
            toggleAllComments(elem)
```
This change is unneeded.

Reply:
Thanks, I dont know how that semi colon got there.